forte/base_classes/active_space_solver.h

```diff
@@ -36,6 +36,11 @@
 #include "psi4/libmints/matrix.h"
 
 #include "base_classes/state_info.h"
+#include <ambit/tensor.h>
+
+namespace ambit {
+class BlockedTensor;
+}
```

Comment:
Move this block to above any forte includes.

Reply:
done